metaworld/__init__.py

```diff
@@ -78,7 +78,7 @@ def _encode_task(env_name, data):
     return Task(env_name=env_name, data=pickle.dumps(data))
 
 
-def _make_tasks(classes, args_kwargs, kwargs_override):
+def _make_tasks(classes, args_kwargs, kwargs_override, seed=0):
```

Comment:
can we make the default value for this np.random.randit(0,1000). If users don't set their seed manually, I'd like it by default be random.